pkg/tests/ossm/smoke_test.go

```diff
@@ -63,6 +126,36 @@ func TestBookinfoInjection(t *testing.T) {
 	})
 }
 
+func assertProxiesReadyInLessThan10Seconds(t TestHelper, ns string) {
+	t.Log("Extracting proxy startup time and last transition time for all the pods in the namespace")
+	startedAt := oc.GetJson(t, ns, "pods", "", `{range .items[*]}{.status.containerStatuses[?(@.name=="istio-proxy")].state.running.startedAt}{"\n"}{end}`)
+	readyAt := oc.GetJson(t, ns, "pods", "", `{range .items[*]}{.status.conditions[?(@.type=="Ready")].lastTransitionTime}{"\n"}{end}`)
```

Comment:
Now I wonder whether the pods are always returned in the same order...? Maybe it would be better to fetch pods, iterate over them and fetch `startedAt` and `readyAt` for each pod individually. Then we would be 100% sure that a give pair of `startedAt` and `readyAt` refers to the same pod. What do you think?

Reply:
Yes, I think will be better. I'll do the change